docs/pages/material-ui/api/backdrop.json

```diff
@@ -19,6 +19,7 @@
     },
     "slots": {
       "type": { "name": "shape", "description": "{ root?: elementType }" },
+      "transition": { "name": "shape", "description": "{ transition?: elementType }" },
```

Comment:
No need to update this manually. Run `pnpm docs:api` instead

Reply:
whoops, my bad

I presume this is the same as lines 32-37

Reply:
Yes! This file and `docs/transations/api-docs/backdrop/backdrop.json` are not to be updated manually, they're updated using the command I shared.

Reply:
> whoops, my bad
> 
> I presume this is the same as lines 32-37

same here :sweat_smile: